flink-runtime/src/main/java/org/apache/flink/runtime/net/NetUtils.java

```diff
@@ -76,7 +76,7 @@ public static InetAddress resolveAddress(InetSocketAddress jobManagerAddress) th
 						case ADDRESS:
 							if (hasCommonPrefix(jobManagerAddress.getAddress().getAddress(), i.getAddress())) {
 								if (tryToConnect(i, jobManagerAddress, strategy.getTimeout())) {
-									LOG.info("Determined " + i + " as the TaskTracker's own IP address");
+									LOG.info("Determined " + i + " as the machine's own IP address");
```

Comment:
Better to use Log.info("Determined {} as the machine's own IP address.", i). That way, the string will only be created if the log level is info or higher.